### PR TITLE
Resprited and GAGS'd Varsity jacket + An additional leather jacket option

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request
Tin. Updates the varsity jacket to use GAGS and resprites it to something bulkier and a bit more referential as well.
Adds a leather coat with a hoodie underneath is, cause that's a kino as fuck look.

Without further ado, do you know what time it is?
image

Leather coat:
image

## How This Contributes To The Skyrat Roleplay Experience
customization or something.

## Changelog

:cl:
imageadd: The Varsity jacket has been GAGSified
imageadd: a new flavor of leather coat has been added to the loadout menu, check there.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 1]
```diff
@@ -503,6 +503,15 @@
...
 	json_config = 'modular_skyrat/modules/GAGS/json_configs/peacoat/peacoat_worn.json'
 
+/datum/greyscale_config/varsity
+	name = "Varsity Jacket"
+	icon_file = 'modular_skyrat/modules/GAGS/icons/suit/suit.dmi'
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/suits/varsity/varsity.json'
+
+/datum/greyscale_config/varsity/worn
+	name = "Varsity Jacket (Worn)"
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/suits/varsity/varsity_worn.json'
```
Comment (ghost) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm:513]: 2 jsons 😭

File: modular_skyrat/modules/customization/modules/clothing/suits/coats.dm [hunk 2]
```diff
@@ -253,3 +253,32 @@
...
+	greyscale_config = /datum/greyscale_config/varsity
+	greyscale_config_worn = /datum/greyscale_config/varsity/worn
+	greyscale_colors = "#553022#a67a5b#2d2d33"
+	body_parts_covered = CHEST|GROIN|ARMS
+	flags_1 = IS_PLAYER_COLORABLE_1
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+
+/obj/item/clothing/suit/hooded/leather
+	name = "hooded leather coat"
+	desc = "A simple leather coat with a hoodie underneath it, not really hooded is it?"
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/suits/coats.dm:270]: don't get the 'not really hooded' comment, it has a hood .-.

**Comment by ghost** [comment]
all checks out, hype hype

File: modular_skyrat/modules/customization/modules/clothing/suits/coats.dm (hunk 2 above)
Comment (Ebin-Halcyon) [reply modular_skyrat/modules/customization/modules/clothing/suits/coats.dm:270]: What was implied was the leather coat itself didn't have a hood, it has a hoodie underneath it.

File: modular_skyrat/modules/GAGS/greyscale_configs.dm (hunk 1 above)
Comment (Ebin-Halcyon) [reply modular_skyrat/modules/GAGS/greyscale_configs.dm:513]: I am lazy :)

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm [hunk 3]
```diff
@@ -149,6 +149,10 @@ GLOBAL_LIST_INIT(loadout_exosuits, generate_loadout_items(/datum/loadout_item/su
 	name = "Biker Jacket"
 	item_path = /obj/item/clothing/suit/jacket/leather/biker
 
+/datum/loadout_item/suit/leather_jacket/hooded
+	name = "Leather Jacket with a hoodie"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]:
```suggestion
	name = "Leather Jacket with a Hoodie"
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Also I know it's a bit annoying, but please add pictures of all orientations when you add worn sprites, as there's sometimes jankiness in certain orientations, or color differences (like the back sprite of one of the jackets seems noticeably darker than the other orientations?)

**Comment by Ebin-Halcyon** [comment]
> Also I know it's a bit annoying, but please add pictures of all orientations when you add worn sprites, as there's sometimes jankiness in certain orientations, or color differences (like the back sprite of one of the jackets seems noticeably darker than the other orientations?)

I didn't edit it much in the color department, it's basically just the base TG leather coat with a hoodie underneath it. And yeah i'll add all 4 dirs in later PRs.

**Review by GoldenAlpharex — APPROVED** [review]
Alright let's go.

**Pull request merged**